cupyx/scipy/sparse/linalg/_eigen.py

```diff
@@ -101,7 +105,7 @@ def eigsh(a, k=6, *, which='LM', ncv=None, maxiter=None, tol=0,
         u -= u.T @ V[:k].conj().T @ V[:k]
         V[k] = u / cublas.nrm2(u)
 
-        u[...] = a @ V[k]
+        u[...] = a_matvec(V[k])
```

Comment:
`LinearOperator` supports `@` operation too.
```suggestion
        u[...] = a @ V[k]
```

Reply:
Thanks you for pointing that out. I didn't know that 😅